src/systemd/sd-dhcp6-option.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#ifndef foosddhcp6optionhfoo
+#define foosddhcp6optionhfoo
+
+/***
+  Copyright © 2013 Intel Corporation. All rights reserved.
```

Comment:
Is the Copyright line necessary? We removed almost all of them because they bring no value over `git log`. If required by company policy, too bad, but if not necessary, we very much prefer not to have them. Also the year seems wrong ;)

Reply:
Nope, that's a cut'n'paste.  I'l remove.